### PR TITLE
Adding Data Sovereignty Section

### PR DESCRIPTION
**Pull request by rastacalavera**
With the recent SSH episodes about photos and data control, I thought we could expand the discussion and how-tos of the wiki

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,4 +7,4 @@ services:
     volumes:
       - .:/docs
     ports:
-      - 8000:8000
```
Comment (BCNelson) [inline docker-compose.yml:10]: I don't think that we want this to be changed

File: mkdocs.yml [hunk 2]
```diff
@@ -46,11 +46,14 @@ markdown_extensions:
...
   - mdx_include:
       base_path: docs
+  - attr_list
+  - md_in_html
 
 plugins:
   - search
   - minify:
       minify_html: true
+  - glightbox
```
Comment (BCNelson) [inline mkdocs.yml:56]: What is this plugin used for?

**Review by BCNelson — CHANGES_REQUESTED** [review]

**Comment by rastacalavera** [comment]
This one?
https://squidfunk.github.io/mkdocs-material/reference/images/?h=light

It makes it so that when you click a picture, it is light boxed and if
there any many included pictures on the same page, the make a gallery or
carousel depending on the setting. Just an enhancement I think it nice.
Found it in the

File: docker-compose.yml (hunk 1 above)
Comment (rastacalavera) [reply docker-compose.yml:10]: Oh I had to change mine cause I had a service on that port already. I can change it back for the pr

Comment (BCNelson) [reply docker-compose.yml:10]: Once this change has been made I will approve and you can merge

Comment (rastacalavera) [reply docker-compose.yml]: Ok I think i changed it back to the appropriate port of 8000

**Pull request merged**